pagopa/io-app
# feat: [IOCOM-453] Add F24 section on SEND message screen

> [!warning]
> This PR depends on #5149 

## Short description
This PR adds support for F24 section on SEND message screen. If we have only one F24, it is rendered in the detail of the message otherwise the "Vedi modelli F24" button is rendered.

### Preview

| === 1 | > 1 |
| --- | --- |
| <img src="https://github.com/pagopa/io-app/assets/29163287/916e080a-0012-4a2b-b2b9-f50dbb4edc82" width="300" /> | <img src="https://github.com/pagopa/io-app/assets/29163287/67876ced-7014-44e7-af78-5854af79e43e" width="300" /> | 

## List of changes proposed in this pull request
- Added F24 section
- Added the new component `MessageAttachments`
- Marked as legacy the old component `MessageAttachments`
- Updated langs   

## How to test
Generate a SEND message with F24 attachemts using `io-dev-api-server`. Check that F24 section is displayed correctly.
